.github/workflows/create-release.yml

```diff
@@ -48,17 +46,6 @@ jobs:
           fetch-depth: 0
           fetch-tags: true
 
-      - name: Exctract Production Readiness
-        run: |
-          if [ -z "$IS_PRODUCTION_READY" ]; then
-            if [[ "${{ github.event_name }}" == "push" && "${{ github.ref_type }}" == "tag" ]]; then
-              echo "IS_PRODUCTION_READY=true" >> $GITHUB_ENV
-            else
-              echo "IS_PRODUCTION_READY=false" >> $GITHUB_ENV
-            fi
-          fi
-          echo "Production readiness determined: $IS_PRODUCTION_READY"
```

Comment:
I think we might need this for latest steps; we should keep that in mind if the rc job fails